ruby/support/app_identity/suite/generator.rb

```diff
@@ -102,120 +102,175 @@ def normalize_test(type, input, index)
     must_have!(type, input, index, "proof")
     must_have!(type, input, index, "spec_version")
 
-    if !%w[pass fail].include?(input["expect"])
+    unless input["expect"] == "pass" || input["expect"] == "fail"
```

Comment:
(nit) I consider using `unless` with anything more complicated than a single term a code smell. I have a weak preference for `if input["expect"] != "pass" && input["expect"] != "fail"`